robyn/__init__.py

```diff
@@ -440,35 +533,35 @@ def configure_authentication(self, authentication_handler: AuthenticationHandler
 
 class SubRouter(Robyn):
     def __init__(self, file_object: str, prefix: str = "", config: Config = Config()) -> None:
```

Comment:
```suggestion
    def __init__(self, file_object: str, prefix: str = "", config: Config = Config(), openapi = OpenAPI()) -> None:
```

This is to avoid coupling.